src/pages/Swap/index.tsx

```diff
@@ -79,10 +94,21 @@ export enum CoWTradeState {
 }
 
 export default function Swap() {
+  const showSimpleChartLocalStorageKey = 'showSimpleChart'
+  // eslint-disable-next-line eqeqeq
+  const getShowSimpleChartPreference = () => window.localStorage.getItem(showSimpleChartLocalStorageKey) == 'true'
```

Comment:
Like we talked about yesterday, I think we can make some variable in the user store with reducer and select and set some enum with all the options, as there will be a Pro tab, and would be nice to be a reusable method

Reply:
Yeah, this is interesting. I'll explore and get back to you.

Reply:
yeah regarding this theres really userful premade hook for local storage that handles everything check it out! https://usehooks-ts.com/react-hook/use-local-storage

Reply:
we are using redux-localstorage-simple, so I would go this way

Reply:
Done.

Reply:
Thanks!